crates/apollo-compiler/src/lib.rs

```diff
@@ -187,7 +168,7 @@ impl ApolloCompiler {
     ///
     /// Returns a `FileId` that you can use to update the source text of this document.
     pub fn add_type_system(&mut self, input: &str, path: impl AsRef<Path>) -> FileId {
-        if self.db.type_system_hir_input().is_some() || self.db.schema_input().is_some() {
+        if self.db.schema_input().is_some() {
```

Comment:
to double check my understanding, we are currently calling:
- `schema` - a hir-based (and programmatic also?) type system representation
- `type_system` - a string-based type system representation

Reply:
It’s not 100% complete with this PR, but the plan is to have v1.0 no longer use `type_system` terminology at all.

`Schema` and `ExecutableDocument` are the two main types. Together we can informally call them "HIR" but that’s also not in public APIs.